roadmap/2023.md

```diff
@@ -0,0 +1,43 @@
+# Roadmap 2023
+
+This is the roadmap planned by the Integration team at Meilisearch for 2023.
+
+Nothing is definitive: the roadmap can be adapted depending on the need of the users and the company.
+
+## 2023: the Main Lines
+
+#### Objectives
+
+- Focus on the impact: Spend time updating SDKs that are mostly used, and work with the community for the other ones.
+- Keep working with the community, improve our way of working with the contributors by creating more evident issues, and provide more and more "good first issues".
+- Work closer to the DevRel and Docs teams to help engage new users and make them stay.
+- Release at least two new integrations/tools based on the data collected by the community.
+- Rework the current DevOps process and make it stable and reliable.
+  - Also add the support for the most significant providers out there, which we are not present yet (Azure and Alibaba cloud)
+
+## Q1
+
+We're entering a new era in 2023, and now we want to achieve some goals we were unable to last year. It is time to increase our DevOps CI's reliability and take the opportunity to be present in two of the biggest cloud providers: Azure and Alibaba. Together they control 26% of the market share. This could be a good advertisement for the company and bring more users.
+
+We are also entering a phase where new projects, not just integrations, will be requested by the team, so we must know where to spend our precious maintenance time. That's why we're going to introduce SDK tiers.
+
+
+Processes:
+- [ ] Introduce the versioning policy for the post-v1 era. (O1K1)
```

Comment:
Where are described O1K1?

Reply:
I tought you were asking that information just for quickly recognition. I didn't think you would like to have the OKR written here. 
I don't think if I share publically the OKRs they will bring much value for the community.

Reply:
But in any case, I added them in the top of the file, check it out 😄